tests/api2/test_020_account.py

```diff
@@ -1,50 +1,33 @@
-#!/usr/bin/env python3
+import pytest
```

Comment:
Thanks! One last thing about this `test_20_account.py` file. We already have a `test_account.py` file that tests quite a few other things. We should remove this file and copy and paste the tests over to that file and re-use existing boilerplate over there if we can.